cmd/apiProxyUpload_test.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRunApiProxyUpload(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Successfull Api Proxy Create Test", func(t *testing.T) {
+		filesMock := mock.FilesMock{}
+		path := filepath.Join("tempDir", "apiproxy.zip")
+		filesMock.AddFile(path, []byte("dummy content"))
```

Comment:
Any way to check that the payload was delivered correctly to the http request?

Reply:
ApiProxyUpload CPIFunction provides proper http response for the /apiportal/api/1.0/Transport.svc/APIProxies call

Reply:
And where is the response checked? Does it include the string "dummy content"?

Reply:
here we are bothering only binary file upload mock, no need of content integrity check